packages/flutter/test/widgets/sliver_tree_test.dart

```diff
@@ -650,6 +650,103 @@ void main() {
     expect(style!.duration, const Duration(milliseconds: 200));
   });
 
+  testWidgets('TreeSliverNode should close all children when collapsed when animation is completed', (WidgetTester tester) async {
```

Comment:
This test passes without the change above. It should fail without the changes to verify they fix the bug.

Reply:
I am trying to make this into test.

Reply:
I tried but I couldn't reproduce the error in the tests.

Reply:
I managed to write the test. I don't think I'm that dumb lol.